lib/modules/auth/login/login_screen.dart

```diff
@@ -1,232 +1,210 @@
-                          validator: (String value) {
-                            if (value.isEmpty) {
-                              return "Password shouldn't be Empty";
-                            } else
-                              return null;
-                          },
-                          decoration: InputDecoration(
-                            labelText: "Password",
-                            prefixIcon: Icon(Icons.lock_outline_rounded),
-                            suffixIcon: Icon(LoginCubit.get(context).suffix),
-                            border: OutlineInputBorder(
-                              borderRadius: BorderRadius.circular(20),
-                            ),
-                          ),
-                        ),
-                        SizedBox(height: 32),
-                        ConditionalBuilder(
-                          condition: state is! LoginLoadingState,
-                          builder: (context) {
-                            return Container(
-                              height: 45,
-                              width: double.infinity,
-                              child: ElevatedButton(
-                                onPressed: () {
-                                  if (formKey.currentState.validate()) {
-                                    LoginModel loginModel = LoginModel(
-                                      email: emailController.text.trim(),
-                                      password: passwordController.text.trim(),
-                                    );
-                                    LoginCubit.get(context).userLogin(
-                                      loginModel: loginModel,
-                                    );
-                                  }
-                                },
-                                child: Text("LOGIN"),
-                              ),
-                            );
-                          },
-                          fallback: (context) => buildLoadingState(),
-                        ),
-                        SizedBox(height: 16),
-                        Row(
-                          mainAxisAlignment: MainAxisAlignment.center,
-                          children: [
-                            Text("Don't have an account?"),
-                            TextButton(
-                                onPressed: () =>
-                                    navigateToRegisterScreen(context),
-                                child: Text(
-                                  "Register",
-                                  style: TextStyle(
-                                      fontWeight: FontWeight.bold,
-                                      fontSize: 14),
-                                )),
-                          ],
-                        )
-                      ],
+    return BlocProvider<LoginBloc>(
+      create: (context) => LoginBloc(LoginRepository()),
```

Comment:
why you create the Login Bloc in the Login Screen not in the main

Reply:
I will just only need it in login so why make it in main?